src/client/utilities.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = {
+  /**
+   * Converts an object of style {'a': {'b': c}, 'd': {'e': f}} to [{'b': c}, {'e': f}]
+   * @param givenObject
+   * @returns {Array}
+   */
+  'flattenObjectToArray': function (givenObject) {
+    let result = [];
+    for (let key in givenObject) {

```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [`key` is never modified, use `const` instead.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2158438033/issues/source?bid=3148782&fileBranchId=3159641#l11)